tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -68,6 +71,19 @@ sub run {
     my $firefox_version = script_output(q(rpm -q MozillaFirefox | awk -F '-' '{ split($2, a, "."); print a[1]; }'));
     record_info('MozillaFirefox version', "Version of Current MozillaFirefox package: $firefox_version");
 
+    if (get_var('FIPS_ENABLED')) {
+        zypper_call('in mozilla-nss mozilla-nss-certs');
+        my $mozilla_nss_ver = script_output("rpm -q --qf '%{version}\n' mozilla-nss");
+
+        # mozilla-nss attempt to update to 3.68+ in SLE15 SP4 base on FIPS certification 140-3
+        if (!is_sle('<15-sp4') && ($mozilla_nss_ver ge 3.68)) {
+            record_info('mozilla_nss version', "Version of Current openssl package: $mozilla_nss_ver");
```

Comment:
BTW, **version of Current openssl package** , openssl may be typo? should be mozilla_nss, correct?

Reply:
@rfan1 yes, this is definitely a typo, thanks for finding :)

Reply:
change to package_upgrade_check()